docker-compose.yml

```diff
@@ -0,0 +1,103 @@
+version: '3.8'
+
+services:
+  webserver:
+    platform: linux/arm64
+    container_name: oppia-webserver
+    build:
+      context: .
+      dockerfile: Dockerfile
+    ports:
+      - "8181:8181"
+    volumes:
+      - .:/app/oppia
+      - node_modules:/app/oppia/node_modules
+    environment:
+      - NODE_ENV=development
+    depends_on:
+      - firebase_emulator
+      - redis
+    command: ./node_modules/.bin/ng build --watch
+  
+  webpack-compiler:
+    platform: linux/arm64
+    container_name: oppia-webpack-compiler
+    build:
+      context: .
+      dockerfile: Dockerfile
+    volumes:
+      - .:/app/oppia
+      - node_modules:/app/oppia/node_modules
+    command: node ./node_modules/webpack/bin/webpack.js --config ./webpack.dev.config.ts --watch
+
+  google_app_engine:
+    build: 
+      context: .
+      dockerfile: Dockerfile.google_cloud_sdk
+    container_name: oppia-app-engine
+    ports:
+      # - "8089:8089"
+      # - "8181:8181"
+      - "8000:8000"
+    volumes:
+      - .:/app/oppia
+    # environment:
+    #   - CLOUDSDK_COMPONENT_MANAGER_DISABLE_UPDATE_CHECK=false
+    depends_on:
+      - webserver
+      - webpack-compiler
+    # restart: unless-stopped
+    # command: ../google-cloud-sdk/bin/dev_appserver.py /app/oppia/app_dev.yaml --host=localhost
+    # command: dev_appserver.py ../app/oppia/app_dev.yaml --host=localhost
+    command: ../google-cloud-sdk/bin/dev_appserver.py /app/oppia/app_dev.yaml --runtime custom --host=localhost
```

Comment:
There seems to be a lot more args here:

https://github.com/oppia/oppia/blob/12f9659179c3d764323d9dbb320f2042ddd7e02e/scripts/servers.py#L182-L195

Reply:
after adding those args, then also same error is occuring